src/test/optimizer/lqp_translator_test.cpp

```diff
@@ -105,6 +118,110 @@ TEST_F(LQPTranslatorTest, PredicateNodeBinaryScan) {
   EXPECT_EQ(table_scan_op->right_parameter(), AllParameterVariant(42));
 }
 
+TEST_F(LQPTranslatorTest, PredicateNodeIndexScan) {
+  /**
+   * Build LQP and translate to PQP
+   */
+  const auto stored_table_node = std::make_shared<StoredTableNode>("table_int_float_chunked");
+
+  const auto table = StorageManager::get().get_table("table_int_float_chunked");
+  std::vector<ColumnID> index_column_ids = {ColumnID{1}};
+  std::vector<ChunkID> index_chunk_ids = {ChunkID{0}, ChunkID{2}};
+  table->get_chunk(index_chunk_ids[0])->create_index<GroupKeyIndex>(index_column_ids);
+  table->get_chunk(index_chunk_ids[1])->create_index<GroupKeyIndex>(index_column_ids);
+
+  auto predicate_node = std::make_shared<PredicateNode>(LQPColumnReference(stored_table_node, ColumnID{1}),
+                                                        PredicateCondition::Equals, 42);
+  predicate_node->set_left_child(stored_table_node);
+  predicate_node->set_scan_type(ScanType::IndexScan);
+  const auto op = LQPTranslator{}.translate_node(predicate_node);
+
+  /**
+   * Check PQP
+   */
+  const auto union_op = std::dynamic_pointer_cast<UnionPositions>(op);
+  ASSERT_TRUE(union_op);
+
+  const auto index_scan_op = std::dynamic_pointer_cast<const IndexScan>(op->input_left());
+  ASSERT_TRUE(index_scan_op);
+  EXPECT_EQ(this->get_included_chunk_ids(index_scan_op), index_chunk_ids);
```

Comment:
`this->`? :(

Reply:
Damn, I saw this in other tests and thought it would be necessary.

Reply:
Other tests USE IT TOO? AAAAAAH.